tests/store/createStore.spec.js

```diff
@@ -3,15 +3,11 @@ import {
 } from 'store/createStore'
 
 describe('(Store) createStore', () => {
-  let store
-
-  before(() => {
-    store = createStore()
-  })
+  let store = createStore()
```

Comment:
this should definitely use `beforeEach`. otherwise the same store will be reused in multiple tests (leaking state).

or, maybe even better, this variable should be created in every test separately